src/atoms/Identicon/Identicon.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import styled from '_styled-components';
+import makeBlockie from 'ethereum-blockies-base64';
+
+const RoundedImage = styled.img`
+  border-radius: 100%; /* stylelint-disable-line unit-whitelist */
+`;
+
+export const Identicon = ({ address }: { address: string }) => {
```

Comment:
Do you think we'll ever want to display an identicon for a hashed string that isn't an address?